addon/components/notification-message.js

```diff
@@ -58,6 +64,11 @@ export default Ember.Component.extend({
     }
   },
 
+  clickableClass: Ember.computed('notification.onClick', function() {
```

Comment:
This property becomes redundant if we include the class in the `classNameBindings` array:

`'notification.onClick:c-notification--clickable'`